managed-os/build.f/particles-managed-os/particles/partials/cf_init.hbs

```diff
@@ -59,15 +62,15 @@
     "routes": {
         "commands": {
             "add-routes": {
-                "command": { "Fn::Join": ["", ["powershell.exe -executionpolicy unrestricted -Command C:\\sungardas\\Windows.ps1 "," -gatewayip ", "{{opsware_address}}"]]},
+                "command": { "Fn::Join": ["", ["powershell.exe -executionpolicy unrestricted -Command C:\\sungardas\\Windows.ps1 "," -gatewayip ", { "Ref": "opswareAddress" }]]},
                 "waitAfterCompletion": 0
             }
         }
     },
     "hpsaagentinstall": {
         "commands": {
-            "1-HPSAAgent-Install": {
-                "command": "C:\\sungardas\\opsware-agent-50.0.59798.1-win32-6.1-X64.exe --opsw_gw_addr {{opsware_address}}:{{opsware_port}}"
+            "HPSAAgent-Install": {
+                "command":  { "Fn::If" : [ "is32BitOs", {"Fn::Join": ["", ["C:\\sungardas\\opsware-agent-win32.exe --opsw_gw_addr ", { "Ref": "opswareAddress" },":{{opsware_port}}"]]},{"Fn::Join": ["", ["C:\\sungardas\\opsware-agent-win64.exe --opsw_gw_addr ",{ "Ref": "opswareAddress" }, ":{{opsware_port}}"]]}]}
```

Comment:
Since we are parameterizing even C:\\sungardas\\opsware-agent-win32.exe --opsw_gw_addr should be parameter based.

So tomorrow if we change the location is just a configuration change